draft-ietf-lamps-csr-attestation.md

```diff
@@ -640,17 +640,21 @@ party that registered the OID.
 ### Initial Registry Contents
 
 The initial registry contents is shown in the table below.
-It lists entries for several evidence encoding including an entry for the Conceptual Message Wrapper (CMW) {{I-D.ietf-rats-msg-wrap}}.
+It lists entries for several evidence encoding OIDs including an entry for the Conceptual Message Wrapper (CMW) {{I-D.ietf-rats-msg-wrap}}.
 
 | OID              | Description                  | Reference(s)     | Change Controller |
 |------------------|------------------------------|----------------  |-------------------|
-| 2 23 133 5 4 1   | DiceTcbInfo                  | {{TCGDICE1.1}}   |  TCG              |
-| 2 23 133 5 4 5   | DiceMultiTcbInfo             | {{TCGDICE1.1}}   |  TCG              |
-| 2 23 133 5 4 6   | DiceUccsEvidence             | {{TCGDICE1.1}}   |  TCG              |
-| 2 23 133 5 4 7   | DiceManifestEvidence         | {{TCGDICE1.1}}   |  TCG              |
-| 2 23 133 5 4 8   | DiceTcbInfoComp              | {{TCGDICE1.1}}   |  TCG              |
-| 2 23 133 5 4 9   | DiceConceptualMessageWrapper | {{TCGDICE1.1}}   |  TCG              |
+| 2 23 133 5 4 1   | tcg-dice-TcbInfo             | {{TCGDICE1.1}}   |  TCG              |
+| 2 23 133 5 4 3   | tcg-dice-endorsement-manifest-uri | {{TCGDICE1.1}}   |  TCG         |
+| 2 23 133 5 4 4   | tcg-dice-Ueid                | {{TCGDICE1.1}}   |  TCG              |
+| 2 23 133 5 4 5   | tcg-dice-MultiTcbInfo        | {{TCGDICE1.1}}   |  TCG              |
+| 2 23 133 5 4 6   | tcg-dice-UCCS-evidence       | {{TCGDICE1.1}}   |  TCG              |
+| 2 23 133 5 4 7   | tcg-dice-manifest-evidence   | {{TCGDICE1.1}}   |  TCG              |
+| 2 23 133 5 4 8   | tcg-dice-MultiTcbInfoComp    | {{TCGDICE1.1}}   |  TCG              |
+| 2 23 133 5 4 9   | tcg-dice-conceptual-message-wrapper | {{TCGDICE1.1}}   |  TCG       |
+| 2 23 133 5 4 11  | tcg-dice-TcbFreshness        | {{TCGDICE1.1}}   |  TCG              |
 | 2 23 133 20 1    | tcg-attest-tpm-certify       | Private Registry |  TCG              |
+| 1 3 6 1 5 5 7 1 TBA | id-pe-cmw                    | {{I-D.ietf-rats-msg-wrap} | IETF     |
```

Comment:
TBA = 35?

Reply:
```suggestion
| 1 3 6 1 5 5 7 1 35 | id-pe-cmw                    | {{I-D.ietf-rats-msg-wrap} | IETF     |
```

I just checked again. There is a [provisional allocation for 35](https://www.iana.org/assignments/smi-numbers/smi-numbers.xhtml#smi-numbers-1.3.6.1.5.5.7.1.35) already.